src/Metric.jl

```diff
@@ -27,23 +27,54 @@ abstract type LorentzMetric <: Metric end
 """
     MetricManifold{M<:Manifold,G<:Metric} <: Manifold
 
-Equip a manifold with a metric. Such a manifold is generally called pseudo-
-or semi-Riemannian. Each `MetricManifold` must implement
-[`local_metric`](@ref).
+Equip a [`Manifold`](@ref) explicitly with a [`Metric`](@ref) `G`. 
 
-# Constructor
+For a Metric Manifold, by default, assumes, that you implement the linear form
+from [`local_metric`](@ref) in order to evaluate the exponential map.
+
+If the corresponding [`Metric`](@ref) `G` yields closed form formulae for e.g.
+the exponential map and this is implemented directly (without solving the ode),
+you of course can still implement that directly.
 
+# Constructor
     MetricManifold(manifold, metric)
-"""
+
+    """
 struct MetricManifold{M<:Manifold,G<:Metric} <: Manifold
     manifold::M
     metric::G
 end
 
-convert(::Type{MT},M::MetricManifold{MT,GT}) where {MT,GT} = base_manifold(M)
-
-is_decorator_manifold(M::MetricManifold) = Val(true)
+is_decorator_manifold(M::MMT) where {MMT <: MetricManifold}= Val(true)
 
+"""
+    is_default_metric(M,G)
+
+indicate, whether the [`Metric`](@ref) `G` is the default metric for
+the [`Manifold`](@ref) `M`. This means that any occurence of
+[`MetricManifold`](@ref)(M,G) where `is_default_metric(M,G) = Val{true}`
+falls back to just be called with `M` such that the [`Manifold`](@ref) `M`
+implicitly has this metric, for example if this was the first one implemented
+or is the one most commonly assumed to be used.
+"""
+is_default_metric(M::Manifold,G::Metric) = Val(false)
+"""
+    is_default_metric(MM)
+
+indicate, whether the [`Metric`](@ref) `G` is the default metric for
```

Comment:
I think in this docstring `MM.G` and `MM.M` should be used instead of just `G` and `M` (because the function only takes one argument called `MM`).

Reply:
Thanks for carefully reading even the docstrings; you're right that makes the statements far clearer.